site/_config.yml

```diff
@@ -6,8 +6,6 @@ description:
 
 baseurl: '/vega-lite' # the subpath of your site, e.g. /blog/
 url: 'https://vega.github.io' # the base hostname & protocol for your site
-twitter:
```

Comment:
It doesn't look like the site actually checked for this slug ( https://github.com/search?q=repo%3Avega%2Fvega-lite%20twitter&type=code ), so removing it without a bsky replacement seems fine.